build-parent/pom.xml

```diff
@@ -746,6 +732,18 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.owasp</groupId>
+                    <artifactId>dependency-check-maven</artifactId>
+                    <version>${owasp-dependency-check-plugin.version}</version>
+                    <configuration>
+                        <!-- Disable Net Analyzer -->
+                        <assemblyAnalyzerEnabled>false</assemblyAnalyzerEnabled>
+                        <nugetconfAnalyzerEnabled>false</nugetconfAnalyzerEnabled>
+                        <nuspecAnalyzerEnabled>false</nuspecAnalyzerEnabled>
+                        <retireJsAnalyzerEnabled>false</retireJsAnalyzerEnabled>
+                    </configuration>
+                </plugin>
```

Comment:
Is it necessary to have it configured in the root pom and here?

Reply:
Hi @aloubyansky, sorry, missed your ping, OK, right now on main, one can only run the check from the individual extensions which won't work with the action, but I can see I can run it from the `extensions` folder, which I believe would require setting a `working-directory` property to point to `extensions` - but I'm not sure it will be equivalent to running it from the root folder, let me experiment a bit later on and I'll get back to you with the update, thanks

Reply:
Hi Alexey @aloubyansky Sorry for a delay, I just run, on this branch, `mvn -Dowasp-report` in the root (not possible if the root pom does not have it configured) and in `extensions` (which is possible on main, for -Dowasp-check for now).
The dependency-check.html is 20 MB if created in the root, and 13 MB if created in extensions.
I think it makes sense to run from the root as it can check more dependencies, for dev tools, etc...
Do you agree ?
Thanks

Reply:
Sure, my question was about the reason why the config is duplicated. If you configure it in the pluginManagement of the root POM then here you could remove the version and the configuration element.

Reply:
@aloubyansky Oh sorry, see what you mean, totally misunderstood your question :-), I'll have a look to clean it up, thanks

Reply:
Hi @aloubyansky I removed the owasp plugin section from the build parent pom, hopefully I did it correctly, thanks.